### PR TITLE
docs: change semver to SemVer

### PR DESCRIPTION
**Pull request by renmu123**
#### Description of Change
lt may be more suitable to use SemVer than semver. For "SemVer" is the Abbreviation of "Semantic Versioning Specification"
reference: https://semver.org/

#### Checklist
- [x] PR description included and stakeholders cc'd
- [ ] `npm test` passes
- [x] relevant documentation is changed or added

#### Release Notes
Notes: no-notes

### CODE REVIEW

File: docs/tutorial/electron-versioning.md [hunk 1]
```diff
@@ -28,19 +28,19 @@ An app developed with `1.8.1` cannot take the `1.8.3` bug fix without either abs
 
 There are several major changes from our 1.x strategy outlined below. Each change is intended to satisfy the needs and priorities of developers/maintainers and app developers.
 
-1. Strict use of semver
+1. Strict use of SemVer
 2. Introduction of semver-compliant `-beta` tags
```
[the remark this answers is not in the document]
Comment (renmu123) [reply docs/tutorial/electron-versioning.md:32]: reference to https://semver.org?

[the remark this answers is not in the document]
Comment (ahallicks) [reply docs/tutorial/electron-versioning.md:32]: Sorry, should've been more specific:

`semver-compliant`
`SemVer-compliant`

**Comment by miniak** [comment]
@renmu123 documentation only changes should not have release notes

**Comment by miniak** [comment]
@renmu123 please fix the linter error:
```
> electron@13.0.0-nightly.20210115 lint:docs-relative-links /home/builduser/project/src/electron
> python ./script/check-relative-doc-links.py

File Location: /home/builduser/project/src/electron/docs/tutorial/electron-versioning.md
	Broken links: #SemVer
Parsed through 208 files within docs directory and its 70 subdirectories.
Found 1 broken relative links.
```

**Review by miniak — APPROVED** [review]

**Review by zcbenz — APPROVED** [review]

**Pull request merged**